types/rails__actioncable/index.d.ts

```diff
@@ -0,0 +1,52 @@
+// Type definitions for @rails/actioncable 6.1
+// Project: https://github.com/rails/rails/tree/main/actioncable/app/javascript/action_cable
+// Definitions by: Davis Namsons <https://github.com/dnamsons>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// tslint:disable-next-line:no-single-declare-module
+declare module '@rails/actioncable' {
```

Comment:
We should just fix this and remove the lint suppression - can you just lift this remove the `declare module '@rails/actioncable' {` and `}` and export everything?

Reply:
I tried doing this at the beginning, but there seems to be an issue with this due to the fact that the scoped module name differs from the naming that `definitely-typed` has. 
If i export all of the declarations manually, i am unable to import them in the test file as it is unable to find the module - `Cannot find module '@rails/actioncable' or its corresponding type declarations.ts(2307)`